crawl-ref/source/tileweb.cc

```diff
@@ -603,7 +603,39 @@ wint_t TilesFramework::_handle_control_message(sockaddr_un addr, string data)
         // (possibly just as a string, like the lua API for this)
         process_command(CMD_GAME_MENU);
     }
-
+    else if (msgtype == "list_worn_armor")
```

Comment:
I don't think it's great to have a new set of message names mapping in an ad hoc way to commands under a different naming scheme. It would be much better to have a single message type with command names used directly as the message data, and then process this in the crawl binary in a less brute-force way. There's already code to map command names (e.g. strings like "CMD_LIST_ARMOUR") to enum values (see `name_to_command` in macro.cc)